infra/src/main/kotlin/fi/fta/geoviite/infra/tracklayout/LayoutAlignmentDao.kt

```diff
@@ -198,7 +198,8 @@ class LayoutAlignmentDao(
     }
 
     @Transactional
-    fun deleteOrphanedAlignments(): List<IntId<LayoutAlignment>> {
+    fun deleteOrphanedAlignments(branch: LayoutBranch): List<IntId<LayoutAlignment>> {
+        assertMainBranch(branch)
```

Comment:
Näiden SQL-muutokset oli Arilla työn alla, vai? Koska kyllähän täällä tarvisi varmaankin oikea branchi etsiä myös kannassa 🤔

Reply:
Joo siis tämän branchin pointti oli tosiaan leikata muutos dao:jen kohdalta ettei mene päällekäin Arin muutosten kanssa. Branchit kuljetellaan front/controller/service kerrosten läpi mutta DAO:ssa heitetään poikkeus jos se on jotain muuta kun main. Myöskin tiketeissä on erikseen nuo SQL-jutut.

https://github.com/finnishtransportagency/geoviite/pull/1247#discussion_r1602798030